clients/client-testextension/build.gradle.kts

```diff
@@ -29,5 +29,7 @@ dependencies {
   api(libs.junit.jupiter.api)
 
   implementation(project(":nessie-client"))
+  api(project(":nessie-multi-env-test-engine"))
```

Comment:
Isn't `Implementation` enough?

Reply:
I'll double check, but IIRC transitive dependencies are required for `javac` in downstream modules.

Reply:
I stand corrected. `implementation` is sufficient here.

I guess `api` was a remnant of earlier experiments when test code referenced `MultiVersionApiTest` directly.